src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -251,25 +252,23 @@ CObjectSchemaItem::CObjectSchemaItem(const Members& members)
 
 void CObjectSchemaItem::RemoveFakeParams(
     SmartObject& Object, const utils::SemanticVersion& MessageVersion) {
-  for (SmartMap::const_iterator it = Object.map_begin();
-       it != Object.map_end();) {
+  for (SmartMap::const_iterator it = Object.map_begin(); it != Object.map_end();
+       ++it) {
     const std::string& key = it->first;
     std::map<std::string, SMember>::const_iterator members_it =
         mMembers.find(key);
-    if (mMembers.end() == members_it
-        // FIXME(EZamakhov): Remove illegal usage of filed in AM
-        && key.compare(connection_key) != 0 && key.compare(binary_data) != 0 &&
-        key.compare(app_id) != 0) {
-      ++it;
+    // FIXME(EZamakhov): Remove illegal usage of filed in AM
```

Comment:
Remove this fixme

Reply:
@alexkutsan , fixed in 9e3c0d23c33cd5ff17ec5796ebd57e1af7a850b4